app/views/handlers/list_edit_question.py

```diff
@@ -24,7 +24,10 @@ def get_next_location_url(self) -> str:
             return url
 
         if first_incomplete_block := self.get_first_incomplete_list_repeating_block_location_for_list_item(
-            repeating_block_ids=self._schema.list_collector_repeating_block_ids,
+            repeating_block_ids=[
```

Comment:
Since `ListRepeatingQuestion` inherits from this, `repeating_block_ids` could be a cached property so you don't have to define this multiple times